### PR TITLE
Adds codestyle env to tox.ini

### PR DESCRIPTION
**Pull request by CyclingNinja**

### CODE REVIEW

**Comment by Cadair** [comment]
Well it's comforting to know this failed as we suspected it would.

**Comment by Cadair** [comment]
Some of these fixes need upstreaming, which will be a fun thing to unravel.

File: tests/conftest.py
Comment (Cadair) [inline tests/conftest.py:1]: This file should be upstreamed.

File: tox.ini
Comment (Cadair) [inline tox.ini:1]: This file should be upstreamed.

File: {{ cookiecutter.package_name }}/docs/conf.py
Comment (Cadair) [inline {{ cookiecutter.package_name }}/docs/conf.py:1]: This file should be upstreamed.

File: {{ cookiecutter.package_name }}/MANIFEST.in
Comment (Cadair) [inline {{ cookiecutter.package_name }}/MANIFEST.in:1]: This file should be upstreamed.

File: {{ cookiecutter.package_name }}/pyproject.toml
Comment (Cadair) [inline {{ cookiecutter.package_name }}/pyproject.toml:1]: This file should be upstreamed.

File: {{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/__init__.py
Comment (Cadair) [inline {{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/__init__.py:1]: This file should be upstreamed.

File: {{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/_dev/scm_version.py
Comment (Cadair) [inline {{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/_dev/scm_version.py:1]: This file should be upstreamed.

File: {{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/example_mod.py
Comment (Cadair) [inline {{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/example_mod.py:1]: This file should be upstreamed.

File: {{ cookiecutter.package_name }}/licenses/BSD2.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) {% now 'utc', '%Y' %}, {{ cookiecutter.author_name }}
+Copyright (C) {% now 'utc', '%Y' %}, {{ cookiecutter.author_name }}
```
Comment (Cadair) [inline {{ cookiecutter.package_name }}/licenses/BSD2.rst]: undo this?

File: {{ cookiecutter.package_name }}/licenses/BSD3.rst [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) {% now 'utc', '%Y' %}, {{ cookiecutter.author_name }}
+Copyright (C) {% now 'utc', '%Y' %}, {{ cookiecutter.author_name }}
```
Comment (Cadair) [inline {{ cookiecutter.package_name }}/licenses/BSD3.rst]: and this?

File: {{ cookiecutter.package_name }}/licenses/BSD2.rst (hunk 1 above)
Comment (Cadair) [inline {{ cookiecutter.package_name }}/licenses/BSD2.rst]:
```suggestion
Copyright (c) {% now 'utc', '%Y' %}, {{ cookiecutter.author_name }}
```

File: {{ cookiecutter.package_name }}/licenses/BSD3.rst (hunk 2 above)
Comment (Cadair) [inline {{ cookiecutter.package_name }}/licenses/BSD3.rst]:
```suggestion
Copyright (c) {% now 'utc', '%Y' %}, {{ cookiecutter.author_name }}
```

File: {{ cookiecutter.package_name }}/tox.ini [hunk 3]
```diff
@@ -6,6 +6,7 @@ envlist =
     py{39,310,311,312}
```
Comment (nabobalis) [inline {{ cookiecutter.package_name }}/tox.ini:6]: We not dropping 39?

**Pull request merged**